src/views/UserEdit/UserForm.js

```diff
@@ -503,6 +503,7 @@ class UserForm extends React.Component {
                   onToggle={this.handleSectionToggle}
                   initialValues={initialValues}
                   patronGroups={formData.patronGroups}
+                  stripes={this.props.stripes}
```

Comment:
Just a matter of style. I would prefer to add 'stripes' to the object desctructuring in L445 - L452. Then you can write `stripes={stripes}` here.